packages/additional-materials/src/documents/additionalMaterials/promptHelpers.ts

```diff
@@ -5,7 +5,10 @@ export const getLessonTranscript = (transcript: string) => {
   return transcript;
 };
 
-const renderCycle = (cycle: LooseLessonPlan["cycle1"], label: string) => {
+const renderCycle = (
```

Comment:
@JBR90 this was the only 'additional' change i had to make due to making cycle3 nullable ... do you think it is safe?